src/Sync.ts

```diff
@@ -0,0 +1,107 @@
+// "use strict";
+
+import "whatwg-fetch";
+// import { TinyEmitter } from "tiny-emitter";
+// import { pino } from "pino";
+
+import { Repository } from "./Evaluate";
+
+import pkg from "../package.json";
+
+const UA = `Node/${pkg.version}`;
+// const logger = pino({
```

Comment:
logs are needed in Node.js SDK as well

Reply:
using `pino`, since it's high-performance and flexible to transport